src/statistics/GlobalStatisticFunction.js

```diff
@@ -0,0 +1,41 @@
+const { GlobalStatistic } = require('../db/entities/GlobalStatistic');
+const { Game } = require('../db/entities/game');
+
+function activePlayers(players, maxDays = 30) {
+  const today = new Date();
+  const activePlayers = [];
+  for (const player of players) {
+    const lastModified = new Date(player._md);
+
+    if (daysFromDate(lastModified, today) <= maxDays) {
+      activePlayers.push(item);
+    }
+  }
+
+  return activePlayers.length;
+}
+
+function daysFromDate(pastDate, today) {
+  var differenceInTime = pastDate.getTime() - today.getTime();
+  return differenceInTime / (1000 * 3600 * 24);
+}
+
+exports.handler = async function (event, context) {
+  console.log(event);
+  // const gameResponse = await Game.query('game', { index: 'ReverseLookupGSI' });
+  const playerResponse = await Game.query('player', {
+    index: 'ReverseLookupGSI'
+  });
+  const authResponse = await Game.query('auth', { index: 'ReverseLookupGSI' });
+  const activePlayers = activePlayers(authResponse.Items);
```

Comment:
Why are you fetching the auth tokens?

The auth SK on the player is working as a way to link accounts between Slack and out app. For the MVP it's not really used

Reply:
The reason was that i wanted to check last login date. But i guess i can do that other ways. And also i dont think lastmodified date will be updated on login, so this will not work👌